.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
-	// See http://go.microsoft.com/fwlink/?LinkId=827846
-	// for the documentation about the extensions.json format
 	"recommendations": [
-		"ms-vscode.vscode-typescript-tslint-plugin"
+		"ms-vscode.vscode-typescript-tslint-plugin",
+		"deepcode.deepcode"
```

Comment:
This is strange. It seems like we recommend to install our package to those, who installed our package.